app/classifier/tasks/drawing-task-details-editor.cjsx

```diff
@@ -4,6 +4,10 @@ AutoSave = require '../../components/auto-save'
 module.exports = React.createClass
   displayName: 'DrawingTaskDetailsEditor'
 
+  propTypes:
```

Comment:
Also (sorry!), I've never found `React.PropTypes.object` to ever be super meaningful. Maybe check out `React.PropTypes.shape` to specify the required property types.